package.json

```diff
@@ -140,7 +142,8 @@
     "scaffold": "./node_modules/.bin/gulp ensureConfig && ./node_modules/.bin/gulp ensureDevProxy",
     "start": "npm run check-env && NODE_ENV=development webpack-dev-server --config ./webpack/webpack.dev.babel.js --hot --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
     "test": "node jest/gen-config.js && NODE_PATH=node_modules jest --config=jest/config.json --no-cache",
-    "testing": "NODE_ENV=testing webpack-dev-server --config ./webpack/webpack.dev.babel.js --progress --colors --content-base ./build --port $npm_package_config_port"
+    "testing": "NODE_ENV=testing webpack-dev-server --config ./webpack/webpack.dev.babel.js --progress --colors --content-base ./build --port $npm_package_config_port",
+    "validate-build": "node scripts/validate-build.js dist/*.js"
```

Comment:
If you added the executable bit to `scripts/validate-build.js`, then you could just remove the `.js` extension and then change this line to just this: `scripts/validate-build dist/*.js` that way it looks and acts more as a binary as you already have `#!/usr/bin/env node` at the top of the file